src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -16,21 +19,27 @@ public class FindCommand extends Command {
     public static final String COMMAND_WORD = "find";
     public static final String COMMAND_WORD_RANDOM_CASE = "FiNd";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all students whose names contain any of "
-            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all students whose:\n"
+            + "1. names contain any of the specified KEYWORDS (case-insensitive)\n"
+            + "Parameters: [n/KEYWORD [MORE_KEYWORDS]]...\n"
+            + "2. schedules contain any of the specified KEYWORDS (case-insensitive)\n"
+            + "Parameters: [d/KEYWORD [MORE_KEYWORDS]]...\n"
+            + "and displays them as a list with index numbers.\n"
+            + "Example: " + COMMAND_WORD + " n/alice bob charlie" + " d/monday tuesday";
```

Comment:
Would it be better to allow finding by only name or only day rather than making it necessary to fill both name and day parameters

Reply:
> Would it be better to allow finding by only name or only day rather than making it necessary to fill both name and day parameters

Sorry I didnt make it clear in the documentation.
find n/
find d/
find n/ d/
are all possible.
The only constraint for the command is at least one of the parameters must be provided

Reply:
Could you suggest how i can rephrase the Message usage?

Reply:
I apologise. I missed the outer brackets in the documentation and have misunderstood what the code was doing. Perhaps for `MESSAGE_NO_NAME_KEYWORDS` and `MESSAGE_NO_SCHEDULE_KEYWORD` could be changed to `MESSAGE_NO_NAME_KEYWORDS_AFTER_PREFIX` and `MESSAGE_NO_SCHEDULE_KEYWORD_AFTER_PREFIX` for clarity.

Reply:
> I apologise. I missed the outer brackets in the documentation and have misunderstood what the code was doing. Perhaps for `MESSAGE_NO_NAME_KEYWORDS` and `MESSAGE_NO_SCHEDULE_KEYWORD` could be changed to `MESSAGE_NO_NAME_KEYWORDS_AFTER_PREFIX` and `MESSAGE_NO_SCHEDULE_KEYWORD_AFTER_PREFIX` for clarity.

Sounds good to me, thank you! Updated accordingly.